FeedBuilder/frmMain.cs

```diff
@@ -81,18 +83,9 @@ private void frmMain_Load(Object sender, EventArgs e)
 			if (!_argParser.ShowGui) Close();
 		}
 
-		private void InitializeFormSettings()
+		private async Task InitializeFormSettings()

```

Comment:
I'll look again. I knew it was sync in the load of the program. But notice it's different when you load a file. I knew that in load we want it to be sync.